tests/ft_rust.vader

```diff
@@ -1,5 +1,11 @@
 Include: include/setup.vader
 
+Before (create src directory):
+  call mkdir('src')
+
+After (cleanup src directory):
+  !rmdir src
+
```

Comment:
Before/After cannot be used in Neomake's tests like that, we have them globally.
But the `build` dir can be assumed to be there for example.